src/middlewared/middlewared/plugins/vm/pci_freebsd.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from middlewared.service import CallError, Service
+from middlewared.utils import run
+
+from .pci_base import PCIInfoBase
+
+
+RE_AMDVI = re.compile(br'vmm\.amdvi\.enable: 1')
```

Comment:
Why the `b` modifier?

Reply:
Iirc We are unable to decode the output of subprocess calls on some machines so the regex match is done without decoding.